.bcr/metadata.template.json

```diff
@@ -0,0 +1,6 @@
+{
+  "homepage": "https://github.com/bazelbuild/rules_cc"
+  "maintainers": [],
```

Comment:
Who should be the maintainer for rules_cc?

Reply:
Google should be the maintainer, no specific person. Do we have an example of this?

Reply:
The metadata.json for rules_cc already exists: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_cc/metadata.json

I guess it's fine to list it as empty right now, there are many other modules that currently don't have a maintainer, we need to clarify the maintenance for them as well.